pkg/liqonet/utils.go

```diff
@@ -132,13 +132,10 @@ func EnableIPForwarding() error {
 }
 
 /* Helper function to get a mask from a CIDR */
-func GetMask(network string) (uint8, error) {
-	_, net, err := net.ParseCIDR(network)
-	if err != nil {
-		return 0, err
-	}
+func GetMask(network string) uint8 {
```

Comment:
Do you introduce the err control somewhere else?

Reply:
This function receives always a network cidr already validated.